mods/cnc/weapons/explosions.yaml

```diff
@@ -37,6 +37,20 @@ UnitExplode:
 		Explosions: poof
 		ImpactSounds: xplobig6.aud
 
+UnitExplodeShip:
+	Warhead@1Dam: SpreadDamage
```

Comment:
Since you only change one value,
```
UnitExplodeShip:
	Inherits: UnitExplode
	Warhead@2Eff: CreateEffect
		Explosions: building
```
would be nicer IMO.